nion/swift/DocumentController.py

```diff
@@ -3023,6 +3023,25 @@ def invoke(self, context: Window.ActionContext) -> Window.ActionResult:
         context = typing.cast(DocumentController.ActionContext, context)
         window = typing.cast(DocumentController, context.window)
         selected_display_item = context.display_item
+        if selected_display_item:
+            window.export_file(selected_display_item)
+        return Window.ActionResult(Window.ActionStatus.FINISHED)
+
+    def is_enabled(self, context: Window.ActionContext) -> bool:
+        context = typing.cast(DocumentController.ActionContext, context)
+        return len(context.display_items) == 1 and context.display_item is not None
+
+
+class BatchExportAction(Window.Action):
```

Comment:
These should appear in alphabetical order, and I'd rename it to `ExportBatchAction` to make sure it's located in the same area of the code as `ExportAction` when alphabetized.